### DISCUSSION TITLE
同一张图竖排和横排文字共同识别

### DISCUSSION THREAD

**Discussion by ltBai** [PaddlePaddle/PaddleOCR#13446] [Q&A]
大家好，我想问下我在官方demo测试的图片是可以识别出竖排文字的，但是用本地的模型就识别不出是什么原因呢。

这是官方的结果：
<img width="960" alt="image" src="https://github.com/user-attachments/assets/2d6efef8-b95e-4e86-bde2-d4997f7aec35">
这是我本地的结果：
<img width="724" alt="image" src="https://github.com/user-attachments/assets/724d9f90-664d-483f-a3d5-f79710e0c6ca">
中间红色框的部分应当是竖排的，可是本地是横排识别的，
这是我的命令：

python tools/infer/predict_system.py --image_dir="PaddleOCR/PaddleOCR/test/00000.jpg" --det_model_dir="../ch_PP-OCRv4_det_infer/" --cls_model_dir="../ch_ppocr_mobile_v2.0_cls_infer/" --rec_model_dir="../ch_PP-OCRv4_rec_infer/" --use_angle_cls=true

**Comment by UserWangZz** [comment]
可以提供一下图片，我尝试一下，是否一致

**Reply by ltBai** [reply]
> 可以提供一下图片，我尝试一下，是否一致

谢谢您，已经解决啦，是因为和官网模型使用有出入，改用server模型就可以了
